gravitee-apim-e2e/api-test/portal/portal-applications.spec.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { afterAll, beforeAll, describe } from '@jest/globals';
+import { ANONYMOUS, forPortal, forPortalAsAdminUser, forPortalAsApiUser, forPortalAsAppUser, forPortalAsSimpleUser } from '@client-conf/*';
+import { created, noContent, unauthorized } from '../../lib/jest-utils';
+import { ApplicationApi } from '@portal-apis/ApplicationApi';
+import { Application } from '@portal-models/Application';
+import { PortalApplicationFaker } from '@portal-fakers/PortalApplicationFaker';
+
+const portalApplicationApiAsAdminUser = new ApplicationApi(forPortalAsAdminUser());
+const portalApplicationApiAsAppUser = new ApplicationApi(forPortalAsAppUser());
+const portalApplicationApiAsApiUser = new ApplicationApi(forPortalAsApiUser());
+const portalApplicationApiAsSimpleUser = new ApplicationApi(forPortalAsSimpleUser());
+const portalApplicationApiAsAnonymous = new ApplicationApi(forPortal({ auth: ANONYMOUS }));
+
+describe('Portal application tests with authorized users', () => {
```

Comment:
In order to keep the same syntax as other tests ;
Maybe we can put the 2 describe in a single one at higher level, for example :

- describe 'Portal application tests'
  - describe 'Portal application tests with authorized users'
  - describe 'Portal application tests with unauthorized users'